src.ts/utils/address.ts

```diff
@@ -98,26 +98,16 @@ export function getHash(hash: Arrayish) {
     return getChecksum(hexlify(hash));
 }
 
-export function deriveAddress(from: string, nonce: Arrayish | BigNumber | number) {
-    if (!from) { throw new Error('missing from address'); }
-    if (!nonce) { throw new Error('missing nonce'); }
-
-    let value = sha256(concat([
-        getAddress(from), stripZeros(hexlify(nonce))
-    ]));
 
-    return getChecksum(value);
-}
-
-export function getMultiSigAddress(from: string, nonce: Arrayish | BigNumber | number) {
+export function getMultiSigAddress(from: string, nonce: number) {
+    console.log("")
     if (!from) { throw new Error('missing from address'); }
     if (!nonce) { throw new Error('missing nonce'); }
     let hexAddress = isHexString(from) ? from : computeHexAddress(from);
     let value = sha256(concat([
-        hexAddress, stripZeros(hexlify(nonce))
+        hexAddress, stripZeros(hexlify(+nonce))
```

Comment:
if you were to add here, you shouldnt add before passing in for the nonce.

Reply:
+ here means converting string to int. Since javascript is not typed language. we need to case it manually. It doesn't increase the number.